.github/workflows/tag.yaml

```diff
@@ -0,0 +1,81 @@
+name: insolar
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build-pack-release:
+    name: build-pack-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@master
+      - name: gather info
+        id: info
+        run: |
+          echo "::set-output name=tag_name::$(echo $GITHUB_REF | cut -d '/' -f3)"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+          echo "::set-output name=go_version::1.12.15"
```

Comment:
The version here doesn't match one in Dockerfile. Can we somehow use something like 1.12.<latest> everywhere?

Reply:
Sure! Updated.

Reply:
we can pass go version as argument to dockerfile and use GHA variable to pass it

Reply:
oh, I see go version is passes as argument to Dockerfile, so it would be nice just set expected Go version (last 1.12.x patch) in Dockerfile